From 95b143cf47743cf3af8e148380fbd18551084a2c Mon Sep 17 00:00:00 2001
Message-ID: <95b143cf47743cf3af8e148380fbd18551084a2c.1724198118.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1724198118.git.sharlatanus@gmail.com>
References: <cover.1724198118.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 20 Aug 2024 22:29:57 +0100
Subject: [PATCH 21/32] gnu: python-stpipe: Update to 0.7.0.

* gnu/packages/astronomy.scm (python-stpipe): Update to 0.7.0.
[arguments] <#:test-flags>: Remove them as all tests now pass without
issues.

Change-Id: I2eadddc3c80b960d99ec899bc8cfd14cec33c117
---
 gnu/packages/astronomy.scm | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 1bb8356346..49b26e0e8b 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3812,13 +3812,13 @@ (define-public python-stdatamodels
 (define-public python-stpipe
   (package
     (name "python-stpipe")
-    (version "0.6.0")
+    (version "0.7.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "stpipe" version))
        (sha256
-        (base32 "0iipbz5ydxxxk44q8ab0ylk7jpxjfhag4vgkhvpj67zs4s45sd8a"))
+        (base32 "0kxm0vllyi3z149q3g3xpfl0c0wpviiyfh55rv0x6zvs0px6c81j"))
        (modules '((guix build utils)))
        (snippet
         '(begin
@@ -3828,10 +3828,6 @@ (define-public python-stpipe
              (("from astropy.extern.configobj import validate") "import validate")
              (("from astropy.extern.configobj.configobj import ") "from configobj import ")
              (("from astropy.extern.configobj.validate import ") "from validate import "))))))
-    (arguments
-     (list
-      ;; See https://github.com/spacetelescope/stpipe/issues/114
-      #:test-flags #~(list "-k" "not test_roman_datamodel")))
     (build-system pyproject-build-system)
     (propagated-inputs
      (list python-asdf
@@ -3839,7 +3835,8 @@ (define-public python-stpipe
            python-crds
            python-stdatamodels))
     (native-inputs
-     (list python-pytest
+     (list ;; python-jwst ; cycle:  python-stpipe -> python-jwst -> python-stpipe
+           python-pytest
            python-pytest-doctestplus
            python-pytest-openfiles
            python-setuptools-scm))
-- 
2.41.0

